src/quo2/components/notifications/activity_log/view.cljs

```diff
@@ -165,6 +165,7 @@
     :as   props}]
   [rn/view
    {:accessibility-label :activity
+    :pointer-events      :none
```

Comment:
just a tiny nit pick, add a comment explaining why pointer events were disabled. 
and  a todo block to remove this code when a proper fix is implemented.